src/auth/service.go

```diff
@@ -22,14 +22,10 @@ type Authorizator interface {
 	CheckAccess(allowedTenants []string) error
 }
 
-// Manager allows managing policies and roles
-type Manager interface {
-	// Role returns role
-	Role(name string) (*entities.Role, error)
-
-	// Roles returns roles
-	Roles() ([]string, error)
-
-	// UserPermissions Extract User Permissions from UserInfo
-	UserPermissions(info *entities.UserInfo) []entities.Permission
+// Roles allows managing policies and roles
```

Comment:
// Roles allows managing permissions and roles

Reply:
fixed